### ISSUE TITLE
When setting custom stdin it should not attempt to reply back with a full string

### ISSUE THREAD

**Issue by AlexDaniel** [perl6/whateverable#34]
Take a look at this log:

```
<AlexDaniel> committable6: stdin http://websitetips.com/art​icles/copy/lorem/ipsum.txt
<committable6> AlexDaniel, Successfully fetched the code from the provided URL.
<committable6> AlexDaniel, https://gist.github.com/15e3​32d3fd7bb9c084b62cdbd2103fd0
```

It should not reply with a link to github. In this case it is probably enough if it just says “OK, custom STDIN is set” or something like that.

**Comment by AlexDaniel** [comment]
Relevant piece of code: https://github.com/perl6/whateverable/blob/master/Whateverable.pm6#L61

**Comment by AlexDaniel** [comment]
Fixed in bbede988f697ed4b0c6598a4d471a1653678b8b2
